pkgs/test/nixpkgs-check-by-name/scripts/run-local.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2016
```

Comment:
I think you don't need this anymore

Reply:
Now in https://github.com/NixOS/nixpkgs/pull/274591/commits/e130ee33a11ae90585e9f9c09b189c3ccfe70632 :)